DMTN-107.tex

```diff
@@ -67,14 +68,18 @@ \subsection{Alert Production}\label{ssec:req_ap}
 
 \subsection{Alert Packet Contents}\label{ssec:req_apc}
 
-The Data Products Definition Document \citeds{LSE-163} specifies that the alert packets include {\em at least} a 12 month history, and cutouts of the template and difference images (Section 3.5.1).
-The 12 month history is not a true requirement, and should be interpreted as applying only after 12 months of LSST operations.
+\citeds{LSE-61} contains a formal requirement that the alert packets include {\em at least} a 12 month history (if available), as well as cutouts of the template and difference images\dmreq{0274}.
+The 12 month history would not be available in LOY1, and the requirement adequately accounts for this, so it would not be violated in LOY1.
 Alerts issued without image cutouts would not violate any requirements, but would be a non-standard packet. 
 
-\citeds{LSE-61} contains a formal requirement that the variability characterization parameters in an alert packet include data collected during {\em at least} the past year\reqparam{diaCharacterizationCutoff}\dmreq{0319}.
+\citeds{LSE-61} contains a formal requirement that the variability characterization parameters in an alert packet be based on data collected during {\em at least} the past year\reqparam{diaCharacterizationCutoff}\dmreq{0319}.
 In LOY1, releasing variability characterization parameters in the alerts that have been calculated with $<1$ year of data would, technically, be in violation of this requirement.
 However, that is unavoidable no matter which option for alert production in LOY1 is adopted.
 
+\citeds{LSE-61} also contains a formal requirement that all DIA sources in the Prompt Products database have a cross-match positional association with the Data Release objects catalog\dmreq{0324}.
```

Comment:
I would say "Data Release Object catalog" to emphasize (by capitalization) that we're referring to a specfic named entity.

Reply:
object --> Object, easy fix